mteb/abstasks/AbsTaskClusteringFast.py

```diff
@@ -91,11 +91,12 @@ class AbsTaskClusteringFast(AbsTask):
         labels: list[str] | list[list[str]]
     """
 
-    max_documents_to_embed = 16_384
-    max_documents_per_cluster = 2048
+    max_fraction_of_documents_to_embed = 0.04
```

Comment:
I would do:

```suggestion
    max_fraction_of_documents_to_embed = 0.04
    max_document_to_embed = None
```

we also need to document these in the abstask

Reply:
if both is None don't downsample.

Reply:
Oh a note here is also that this will influence existing clustering tasks (some of which are downsampled before they are pushed to the hub). For these just using the current number is probably the best solution.